x/concentrated-liquidity/lp.go

```diff
@@ -80,7 +82,7 @@ func (k Keeper) createPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddr
 	}
 
 	// Update the position in the pool based on the provided tick range and liquidity delta.
-	actualAmount0, actualAmount1, err := k.updatePosition(cacheCtx, poolId, owner, lowerTick, upperTick, liquidityDelta, joinTime, positionId)
+	actualAmount0, actualAmount1, err := k.updatePosition(cacheCtx, poolId, owner, lowerTick, upperTick, liquidityDelta, joinTime, positionId, 0)
```

Comment:
nit: we should have this default value defined as a constant

Reply:
https://github.com/osmosis-labs/osmosis/pull/4854/commits/7857283ffd67c3a3a39ada78f78a2f61a1220c63